sdks/java/io/hadoop-input-format/src/main/java/org/apache/beam/sdk/io/hadoop/inputformat/HadoopInputFormatIO.java

```diff
@@ -234,6 +235,7 @@
     }
 
     /** Reads from the source using the options provided by the given configuration. */
+    @SuppressWarnings("unchecked")
```

Comment:
Can we scope this (and others which are necessary) to just the line which is causing them, rather than putting them at the method level?

Reply:
I can do.  I'm cleaning many IO, and opted to put them on methods when there was >1 usages within the method.  In some IO I found 8 usages within one method which I felt annoyingly cluttered. There are two in this case.

Do you want them on a per line basis? Happy to change for this review.